### PR TITLE
Fix/uno 521/test authoring undo deletion of test element

### PR DESCRIPTION
**Pull request by AndreyNegrov**
__task:__ https://oat-sa.atlassian.net/browse/UNO-521
__description:__ Test Authoring - Undo deletion of a test element
__cause__: popup styles styles overlap  oat-sa / tao-core-ui / scss / inc / _feedback.scss: 6

### CODE REVIEW

**Review by habemuscode — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

File: src/component/scss/components.scss [hunk 1]
```diff
@@ -45,10 +45,10 @@
 
 .undobox {
     display : block;
-    position : fixed;
+    position : fixed!important;
```
Comment (zagovorichev) [inline src/component/scss/components.scss:48]: 1. I don't like this solution with `!important` usage. probably changing scss file ordering make a trick: if you load _feedback.scss and then component.scss
2. when you creating new version of the fe package you have to compile css file, that should appear in the dist directory (your case is dist/component/css/component.css)

**Review by zagovorichev — CHANGES_REQUESTED** [review]
I guess that there is more global issue of the style applying

File: src/component/scss/components.scss (hunk 1 above)
Comment (AndreyNegrov) [reply src/component/scss/components.scss]: done

**Review by zagovorichev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**